roles/kubernetes/secrets/files/make-ssl.sh

```diff
@@ -82,10 +82,13 @@ gen_key_and_cert() {
 
 # Admins
 if [ -n "$MASTERS" ]; then
-    # If any host requires new certs, just regenerate all master certs
     # kube-apiserver
-    gen_key_and_cert "apiserver" "/CN=kube-apiserver"
-    cat ca.pem >> apiserver.pem
+    # keep if ca and apiserver certs exist
+    if ! [ -e "$SSLDIR/ca-key.pem" ] || ! [ -e "$SSLDIR/apiserver-key.pem" ]; then
+      gen_key_and_cert "apiserver" "/CN=kube-apiserver"
+      cat ca.pem >> apiserver.pem
```

Comment:
Why are we overwriting the apiserver.pem?

Reply:
It's actually the opposite. I can change the comment to make it more obvious

Reply:
ignore, misread append as overwrite.